tasks/collect.json

```diff
@@ -1,5 +1,6 @@
 {
   "description": "Collect data for the Puppet Health Check Lite service",
   "parameters": {},
-  "supports_noop": false
+  "supports_noop": false,
+  "files": ["healthcheck_lite/files/common.sh"]
```

Comment:
Is this compatible with Puppet Task in the versions of PE this supports?

Also, we should update metadata.json to document the versions of PE this supports in `dependencies`

Reply:
It does work with `puppet task`, yes.

How is this dependency normally specified?  I see this in the metadata:

```
  "requirements": [
    {
      "name": "puppet",
      "version_requirement": ">= 5.3.2 < 7.0.0"
    }
  ]
```

This is the same as the metadata for `support_tasks`, but we can use this on older versions without `puppet task` now.

Reply:
Ah, its inferred from the Puppet Version:

https://puppet.com/docs/puppet/6.4/modules_metadata.html#specifying-puppet-version-requirements

Reply:
But I think this is valid:

    {
      "name": "pe",
      "version_requirement": ">= 2016.4.0"
    }